packages/amplify-category-auth/src/provider-utils/awscloudformation/question-factories/core-questions.ts

```diff
@@ -1,22 +1,36 @@
-const inquirer = require('inquirer');
-const { uniq, flatten } = require('lodash');
-const chalk = require('chalk');
+/* eslint-disable no-param-reassign */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import inquirer, { QuestionCollection } from 'inquirer';
+import { uniq, flatten } from 'lodash';
+import chalk, { Chalk } from 'chalk';
+import { $TSAny, $TSContext } from 'amplify-cli-core';
 
-function parseInputs(input, amplify, defaultValuesFilename, stringMapsFilename, currentAnswers, context) {
+/**
+ * parses input object and returns a question object
+ */
+export const parseInputs = async (
+  input: $TSAny,
+  amplify: $TSAny,
+  defaultValuesFilename: $TSAny,
+  stringMapsFilename: $TSAny,
+  currentAnswers: $TSAny,
```

Comment:
are there better types for these ? (and other places we use $TSAny below ?)

Reply:
I converted a few types but was unfamiliar with the area and aren't self-evident what it should be here. I also will need to refactor this method in the following PR since these return a set of questions for the inqurier library, which is replaced with the prompter.